basic-javascript/project-1.spec.js

```diff
@@ -1,3 +1,224 @@
+        expect(typeof num).toBe("number");
+    });
+    it("should multiply the num by 10", () => {
+        const multiplyByTen = helpers.multiplyByTen;
+        const expected = multiplyByTen(3);
+        expect(expected).toBe(30);
+    });
+});
+
+describe("subtractFive", () => {
+    it("num should be a number", () => {
+        const num = helpers.subtractFive(3);
+        expect(typeof num).toBe("number");
+    });
+    it("should substract the num", () => {
+        const substractFive = helpers.subtractFive;
+        const expected = substractFive(9);
+        expect(expected).toBe(4);
+    });
+});
+
+describe("areSameLength", () => {
+    const areSameLength = helpers.areSameLength;
+    const str1 = "apple";
+    const str2 = "orange";
+    const str3 = "juice";
+    it("should return a bool", () => {
+        expect(typeof areSameLength(str1, str2)).toBe("boolean");
+    });
+    it("should return true", () => {
+        expect(areSameLength(str1, str3)).toBeTruthy();
+    });
+    it("should return false", () => {
+        expect(areSameLength(str1, str2)).toBeFalsy();
+    });
+});
+
+describe("areEqual", () => {
+    const areEqual = helpers.areEqual;
+    const num1 = 8;
+    const num2 = 5;
+    const num3 = 8;
+
+    it("should return a boolean", () => {
+        expect(typeof areEqual(num1, num2)).toBe("boolean");
+    });
+    it("should return true", () => {
+        expect(areEqual(num1, num3)).toBeTruthy();
+    });
+    it("should return false", () => {
+        expect(areEqual(num1, num2)).toBeFalsy();
+    });
+});
+
+describe("lessThanNinety", () => {
+    const lessThanNinety = helpers.lessThanNinety;
+    it("checks if the num is less than ninety", () => {
+        expect(lessThanNinety(70)).toBeTruthy();
+        expect(lessThanNinety(99)).toBeFalsy();
+    });
+});
+
+describe("greaterThanFifty", () => {
+    const greaterThanFifty = helpers.greaterThanFifty;
+    it("checks if the num is greater than fifty", () => {
+        expect(greaterThanFifty(71)).toBeTruthy();
+        expect(greaterThanFifty(40)).toBeFalsy();
+    });
+});
+
+describe("add", () => {
+    const add = helpers.add;
+    it("should add two numbers", () => {
+        expect(add(8, 2)).toBe(10);
+        expect(add(2, -2)).toBe(0);
+    });
+});
+
+describe("subtract", () => {
+    const subtract = helpers.subtract;
+    it("should substract a number from another number", () => {
+        expect(subtract(9, 3)).toBe(6);
+        expect(subtract(4, 7)).toBe(-3);
+    });
+});
+
+describe("divide", () => {
+    const divide = helpers.divide;
+    it("should divide a number to another", () => {
+        expect(divide(14, 7)).toBe(2);
+        expect(divide(8, 4)).toBe(2);
+    });
+});
+
+describe("multiply", () => {
+    const multiply = helpers.multiply;
+    it("should multiply two numbers ", () => {
+        expect(multiply(3, 7)).toBe(21);
+        expect(multiply(1, -3)).toBe(-3);
+    });
+});
+
```

Comment:
What happens if the user enters something else than a number? Solely to explore tests like .toBeNaN();